run/gem5art/run.py

```diff
@@ -564,12 +664,29 @@ def getRunsByNameLike(db: ArtifactDB, name: str, fs_only: bool = False,
     """
 
     if not fs_only:
-        seruns = db.searchByLikeNameType(name, 'gem5 run', limit=limit)
+        seruns = db.searchByLikeNameType(name, "gem5 run", limit=limit)
 
         for run in seruns:
             yield gem5Run.loadFromDict(run)
 
-    fsruns = db.searchByLikeNameType(name, 'gem5 run fs', limit=limit)
+    fsruns = db.searchByLikeNameType(name, "gem5 run fs", limit=limit)
 
     for run in fsruns:
         yield gem5Run.loadFromDict(run)
+
+
+def getRerunnableRunsByNameLike(
+    db: ArtifactDB, name: str, fs_only: bool = False, limit: int = 0
+) -> Iterable[gem5Run]:
+
+    """ Returns a generator of gem5Run objects having `rerunnable == True`
+    and the object "name" containing the name parameter as a substring. The
+    parameter is case sensitive.
+
+    If fs_only is True, then only full system runs will be returned.
+    Limit specifies the maximum number of runs to return.
+    """
+
+    for run in getRunsByNameLike(db, name, fs_only, limit):
+        if run.rerunnable:
```

Comment:
This could be part of the query in the line above. Though, now that I look at it, I'm not sure that API exists in the db... You could query `{'name':  name, 'rerunnable': True}` or something like that.

Reply:
I think, in general, we can always have attributes as a parameter that this function would return runs having these attributes.

However, at the same time, I think that rerunning stuff is an important feature that should have its own function.